packages/next/src/lib/eslint/getESLintPromptValues.ts

```diff
@@ -1,7 +1,7 @@
 import findUp from 'next/dist/compiled/find-up'
 
 export const getESLintStrictValue = async (cwd: string) => {
-  const hasTSConfig = !!(await findUp('tsconfig.json'), { cwd })
+  const hasTSConfig = !!(await findUp('tsconfig.json', { cwd }))
```

Comment:
🤦

Reply:
Simple code is easier to read and harder to make mistakes on: [`7c99396` (#52845)](https://github.com/vercel/next.js/pull/52845/commits/7c9939695dd2cf92405409564739efbcf6b9a1db)